src/games/strategy/triplea/delegate/MoveDelegate.java

```diff
@@ -95,56 +95,63 @@ public void start() {
         final HashSet<TriggerAttachment> toFirePossible = TriggerAttachment.collectForAllTriggersMatching(
             new HashSet<PlayerID>(Collections.singleton(m_player)), moveCombatDelegateAllTriggerMatch, m_bridge);
         if (!toFirePossible.isEmpty()) {
-          // collect conditions and test them for ALL triggers, both those that we will first before and those we will
+
+          // collect conditions and test them for ALL triggers, both those that we will fire before and those we will
           // fire after.
           testedConditions = TriggerAttachment.collectTestsForAllTriggers(toFirePossible, m_bridge);
           final HashSet<TriggerAttachment> toFireBeforeBonus =
               TriggerAttachment.collectForAllTriggersMatching(new HashSet<PlayerID>(Collections.singleton(m_player)),
                   moveCombatDelegateBeforeBonusTriggerMatch, m_bridge);
           if (!toFireBeforeBonus.isEmpty()) {
+
             // get all triggers that are satisfied based on the tested conditions.
             final Set<TriggerAttachment> toFireTestedAndSatisfied = new HashSet<TriggerAttachment>(
                 Match.getMatches(toFireBeforeBonus, AbstractTriggerAttachment.isSatisfiedMatch(testedConditions)));
+
             // now list out individual types to fire, once for each of the matches above.
             TriggerAttachment.triggerNotifications(toFireTestedAndSatisfied, m_bridge, null, null, true, true, true,
                 true);
             TriggerAttachment.triggerPlayerPropertyChange(toFireTestedAndSatisfied, m_bridge, null, null, true, true,
                 true, true);
             TriggerAttachment.triggerRelationshipTypePropertyChange(toFireTestedAndSatisfied, m_bridge, null, null,
                 true, true, true, true);
-            TriggerAttachment.triggerTerritoryPropertyChange(toFireTestedAndSatisfied, m_bridge, null, null, true, true,
-                true, true);
-            TriggerAttachment.triggerTerritoryEffectPropertyChange(toFireTestedAndSatisfied, m_bridge, null, null, true,
+            TriggerAttachment.triggerTerritoryPropertyChange(toFireTestedAndSatisfied, m_bridge, null, null, true,
                 true, true, true);
+            TriggerAttachment.triggerTerritoryEffectPropertyChange(toFireTestedAndSatisfied, m_bridge, null, null,
```

Comment:
Side comment, the 'null null, true true, true , true' would be sweet to one day get rid of. Looks like a good place for the builder pattern.